CONTRIBUTING.md

```diff
@@ -661,6 +661,11 @@ If you have any problems with formatting, please ping the [formatting team](http
   As an exception, an explicit conditional expression with null can be used when fixing a important bug without triggering a mass rebuild.
   If this is done a follow up pull request _should_ be created to change the code to `lib.optional(s)`.
 
+- Any style choices not covered here should be left at the discretion of the authors of changes and _not_ commented in reviews.
```

Comment:
I'm not so sure we should really ban commenting on style (outside of the written rules) altogether?

Such comments must be clearly communicated as non-blocking, and we should encourage reviewers to consider not leaving comments about things that don't matter, and to approve PRs on which there are some non-blocking nits but are nevertheless valid to be merged. 

But I've received some valuable feedback about "style" things that are not in the documentation! Sometimes, because they were specific to my code instead of a general rule, they should not have been in the documentation. So I'm worried this wording, by preventing that, sort of throws the baby out with the bathwater.